xray/plot/plot.py

```diff
@@ -386,8 +390,12 @@ def _plot2d(plotfunc):
         The mapping from data values to color space. If not provided, this
         will be either be ``viridis`` (if the function infers a sequential
         dataset) or ``RdBu_r`` (if the function infers a diverging dataset).
-        When ``levels`` is provided and when `Seaborn` is installed, ``cmap``
-        may also be a `seaborn` color palette or a list of colors.
+        When when `Seaborn` is installed, ``cmap`` may also be a `seaborn` 
```

Comment:
Now the docs make it clear that you need to specify `levels` (or use `contour`/`contourf`) with a seaborn named palette in `cmap`. That is a slightly different case than specifying a list of `colors`. This seems consistent, because now you never give a list to cmap.